portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +31,131 @@ function addRandomGreeting() {
+        isHatesDisplayed = false;
+    }
+    else {
+        aboutMeContainer.style.display = 'none';
+        isHobbiesDisplayed = false;
+    }
+    text = text = "<ul class=\"collection with-header z-depth-3\"><li class=\"collection-header center purple lighten-4 white-text\"><h4>My Hobbies</h4></li>";;
+    for (i = 0; i < hobbies.length; i++) {
+        text += "<li class=\"collection-item grey lighten-4\">" + hobbies[i] + "</li>"
+    }
+    text += "</ul>"
+    aboutMeContainer.innerHTML = text;
+    
+}
+
+/**
+ * Display what I love
+ */
+function getLoves() {
+    const loves = [
+        'dogs (especially corgis, and I have one!)',
+        'movies/series (favourite genre: romantic & comedy)',
+        'books (favourite genre: romance & self-help)',
+        'fashion',
+        'my planner (keep me organised!)'
+    ];
+
+    //Add them to the page
+    const aboutMeContainer = document.getElementById('about-me-container');
+    if (aboutMeContainer.style.display === 'none') {
+        aboutMeContainer.style.display = 'block';
+        isLovesDisplayed = true;
+    }
+    else if(aboutMeContainer.style.display !== 'none' && (isHobbiesDisplayed === true || isHatesDisplayed === true)) {
+        aboutMeContainer.style.display = 'block';
+        isLovesDisplayed = true;
+        isHobbiesDisplayed = false;
+        isHatesDisplayed = false;
+    }
+    else {
+        aboutMeContainer.style.display = 'none';
+        isLovesDisplayed = false;
+    }
+    text = text = "<ul class=\"collection with-header z-depth-3\"><li class=\"collection-header center purple lighten-4 white-text\"><h4>What I love</h4></li>";;
+    for (i = 0; i < loves.length; i++) {
+        text += "<li class=\"collection-item grey lighten-4\">" + loves[i] + "</li>"
+    }
+    text += "</ul>"
+    aboutMeContainer.innerHTML = text;
+}
+
+/**
+ * Display what I hate
+ */
+function getHates() {
+    const hates = [
+        'mess (so I\'m usually quite organised!)'
+    ];
+
+    //Add them to the page
+    const aboutMeContainer = document.getElementById('about-me-container');
+    if (aboutMeContainer.style.display === 'none') {
+        aboutMeContainer.style.display = 'block';
+        isHatesDisplayed = true;
+    }
+    else if(aboutMeContainer.style.display !== 'none' && (isHobbiesDisplayed === true || isLovesDisplayed === true)) {
+        aboutMeContainer.style.display = 'block';
+        isHatesDisplayed = true;
+        isHobbiesDisplayed = false;
+        isLovesDisplayed = false;
+    }
+    else {
+        aboutMeContainer.style.display = 'none';
+        isHatesDisplayed = false;
+    }
+    text = text = "<ul class=\"collection with-header z-depth-3\"><li class=\"collection-header center purple lighten-4 white-text\"><h4>What I hate</h4></li>";;
+    for (i = 0; i < hates.length; i++) {
+        text += "<li class=\"collection-item grey lighten-4\">" + hates[i] + "</li>"
+    }
+    text += "</ul>"
+    aboutMeContainer.innerHTML = text;
+}
```

Comment:
add a blank new line at the end of files, (sorry I can't find the part of the styleguide that says this, but I promise you it's a thing, even github is telling me you're missing a newline)